threagile.yaml

```diff
@@ -1037,6 +1121,8 @@ technical_assets:
       - manifest-sources
     data_assets_stored:
       - manifest-sources
+    data_assets_produced:
+      - argocd-database-export
```

Comment:
I think this would actually be produced by the API server. I don't think the repo-server is involved in `argocd admin export`'s collection of info.

Reply:
Thanks - moved to APi Server.